tests/wamr-test-suites/spec-test-script/all.py

```diff
@@ -155,7 +155,7 @@ def test_case(
         CMD,
         bufsize=1,
         stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
+        stderr=subprocess.DEVNULL,
```

Comment:
if so, we won't receive error information, and it may limit our choice.

Reply:
do you mean it's better to keep the unused stderr plumbing?